presto-cli/src/main/java/io/prestosql/cli/Console.java

```diff
@@ -170,7 +170,13 @@ private String getPassword()
 
         java.io.Console console = System.console();
         if (console == null) {
-            throw new RuntimeException("No console from which to read password");
+            try {
+                jline.console.ConsoleReader consoleReader = new jline.console.ConsoleReader(System.in, System.err);
+                return consoleReader.readLine("Password: ", Character.valueOf((char) 0));
```

Comment:
This should work without the explicit boxing to `Character`:
```java
return consoleReader.readLine("Password: ", (char) 0);
```

Reply:
Applied